src/plugins/plot/chart/MctChart.vue

```diff
@@ -27,13 +27,13 @@
     <div ref="limitArea" class="js-limit-area">
       <limit-label
         v-for="(limitLabel, index) in visibleLimitLabels"
-        :key="index"
+        :key="`limitLabel-${index}`"
```

Comment:
I don't think that's necessary. `limitLabels` are objects more than 1 level deep.

Reply:
We should prefer to use unique keys instead of array indexes for v-for lists to prevent the case of Vue losing track of items due to array mutations, if there's a straightforward way to do that here.

Reply:
I think i see a way! Take a look.